ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/HomeScreenMapContainerFragment.kt

```diff
@@ -105,8 +103,8 @@ class HomeScreenMapContainerFragment : AbstractMapContainerFragment() {
       }
 
       // Bind data for cards
-      mapContainerViewModel.getMapCardUiData().launchWhenStartedAndCollect { (mapCards, loiCount) ->
-        adapter.updateData(canUserSubmitData, mapCards, loiCount - 1)
+      mapContainerViewModel.getMapCardUiData().launchWhenStartedAndCollect { (loiCard, jobCards) ->
```

Comment:
"Cards" no longer applies here and throughout - we'll now have a button and a bottom sheet. Please rename/refactor accordingly, and wrap the LOI card in a [bottom sheet](https://developer.android.com/develop/ui/compose/components/bottom-sheets).

Reply:
For this PR a card is also fine for the selected LOI, I would just update the "add LOI" button to be an actual button.

Reply:
Done.